engines/director/util.h

```diff
@@ -130,6 +130,17 @@ inline void lerpPalette(byte *target, byte *palA, int palALength, byte *palB, in
 	}
 }
 
+template <typename T> void setFlagBit(T *flags, int position, bool value) {
```

Comment:
I see you use it only with `uint32`. Maybe remove these templates and just use plain functions?